Dockerfile

```diff
@@ -5,6 +5,9 @@ WORKDIR /usr/src/app
 ENV PORT 3122
 EXPOSE 3122
 
+ENV RAYGUN_EXPRESS_SERVER_API_KEY=${RAYGUN_EXPRESS_SERVER_API_KEY}
```

Comment:
Wonder if there should be ARG statements here too.

Reply:
resolved in 0128450 :)